apis/placement/v1alpah1/override_types.go

```diff
@@ -0,0 +1,186 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package v1alpah1
+
+import (
+	placementv1beta1 "go.goms.io/fleet/apis/placement/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +genclient:nonNamespaced
+// +kubebuilder:object:root=true
+// +kubebuilder:resource:scope="Cluster",categories={fleet,fleet-placement}
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ClusterResourceOverride defines a group of override policies about how to override the selected cluster scope resources
+// to target clusters.
+type ClusterResourceOverride struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// The desired state of ClusterResourceOverrideSpec.
+	// +required
+	Spec ClusterResourceOverrideSpec `json:"spec"`
+}
+
+// ClusterResourceOverrideSpec defines the desired state of the Override.
+type ClusterResourceOverrideSpec struct {
```

Comment:
Discuss the conflict-resolving behavior here?

Conflict with another CRO. Conflict with RO.

Reply:
Same for RO.

Reply:
Could you also add what happens if one resource is selected by two CRO?